documentation/guide-how-to-create-a-devon4quarkus-app.asciidoc

```diff
@@ -0,0 +1,213 @@
+
+To load a new Quarkus application from command line with the Quarkus Maven plugin run the following command:
+
+.Operating system
+. Linux and MacOs +
+`mvn io.quarkus:quarkus-maven-plugin:2.0.2.Final:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=getting-started \
+    -DclassName="org.acme.getting.started.GreetingResource" \
+    -Dpath="/hello"`
+
+. Windows +
+    `mvn io.quarkus:quarkus-maven-plugin:2.0.2.Final:create "-DprojectGroupId=org.acme" "-DprojectArtifactId=getting-started" "-DclassName=org.acme.getting.started.GreetingResource" "-Dpath=/hello"`
+ 
+
+it generates the following in the getting-started folder
+
+* the Maven structure
+* an org.acme.getting.started.GreetingResource resource exposed on /hello
+* an associated unit test
+* a landing page that is accessible on http://localhost:8080 after starting the application
+* example Dockerfile files for both native and JVM modes in src/main/docker
+* the application configuration file
+
+Once generarted change your directory with `cd getting-started`and take a look at the *pom.xml*. You will find the import of the Quarkus BOM, allowing you to omit the version on the different Quarkus dependencies. In addition, you can see the *quarkus-maven-plugin* responsible for packaging of the application and also providing the development mode.
+
+[source, xml]
+----
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-universe-bom</artifactId>
+            <version>${quarkus.platform.version}</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+
+<build>
+    <plugins>
+        <plugin>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-maven-plugin</artifactId>
+            <version>${quarkus-plugin.version}</version>
+            <extensions>true</extensions>
+            <executions>
+                <execution>
+                    <goals>
+                        <goal>build</goal>
+                        <goal>generate-code</goal>
+                        <goal>generate-code-tests</goal>
+                    </goals>
+                </execution>
+            </executions>
+        </plugin>
+    </plugins>
+</build>
+----
+
+==== Extension management
+Extensions added by creating a project from https://code.quarkus.io or with the parameter `-Dextensions` in your mvn create command and can be found after that in the pom.xml. Now we are just using *RESTEasy*. 
+
+[source, xml]
+----
+    <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-resteasy</artifactId>
+    </dependency>
+----
+
+Later we will need an additional extension. Add an extension to an already created project by inserting the dependencies manually into the pom.xml or you can also use the mvn command `mvn quarkus:add-extension -Dextensions="container-image-docker"`.
```

Comment:
Are you sure this extension is needed?

Reply:
I needed it for another way to show what the DevUI is capable of and to build a native executable.